src/modals/TestSource/TestSource.vue

```diff
@@ -0,0 +1,166 @@
+<script setup>
+import { sourceStore, navigationStore } from '../../store/store.js'
+</script>
+
+<template>
+	<NcModal v-if="navigationStore.modal === 'testSource'"
+		ref="modalRef"
+		label-id="testSource"
+		@close="closeModal">
+		<div class="modalContent">
+			<h2>Test source</h2>
+
+			<form @submit.prevent="handleSubmit">
+				<div class="form-group">
+					<div class="detailGrid">
+						<NcSelect
+							id="method"
+							v-bind="methodOptions"
+							v-model="methodOptions.value" />
+
+						<NcSelect
+							id="type"
+							v-bind="typeOptions"
+							v-model="typeOptions.value" />
+
+						<NcTextField
+							id="endpoint"
+							label="Endpoint"
+							:value.sync="testSourceItem.endpoint" />
+					</div>
+					<NcTextArea
+						id="body"
+						label="Body"
+						:value.sync="testSourceItem.body" />
+				</div>
+			</form>
+
+			<NcButton
+				:disabled="loading"
+				type="primary"
+				@click="testSource()">
+				<template #icon>
+					<NcLoadingIcon v-if="loading" :size="20" />
+					<Sync v-if="!loading" :size="20" />
+				</template>
+				Test connection
+			</NcButton>
+
+			<NcNoteCard v-if="sourceStore.sourceTest && sourceStore.sourceTest.response.statusCode.toString().startsWith('2')" type="success">
```

Comment:
dit is helemaal niet nodig

`response.ok` doet exact hetzelfde
https://developer.mozilla.org/en-US/docs/Web/API/Response/ok
![image](https://github.com/user-attachments/assets/b1a6fe47-0879-4c0b-9763-311003fc6c54)